ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1177,6 +1177,19 @@ SWTC:
   - supply_water_temperature_sensor
   - supply_water_temperature_setpoint
 
+# Rename SFN TOTAL_
+SATC:
```

Comment:
This is generally discouraged; *how* does it control the supply temperature? With a heater? A DX unit? A chilled water coil?

Reply:
@cstirdivant - sorry for delay, This AHU is provided a cooling_percentage command by the DX Unit. I'm struggling to add on the cooling_percentage_command. Can you give me any advice on adding it to the Entity Type. Thanks a million, Ciaran

Reply:
Hi @Ciaran-Oflannagain I would use a full DX abstract type (e.g., DXSC) and mark whatever required points as MISSING (oif your model does not expose them).